sys/include/div.h

```diff
@@ -33,17 +33,42 @@ extern "C" {
 /**
  * @brief Approximation of (2**l)/d for d=15625, l=12, 32 bits
  */
-#define DIV_H_INV_15625_32    0x431bde83ul
+#define DIV_H_INV_15625_32     0x431bde83ul
```

Comment:
ah haven't looked into the existing code. mhm then I think its better to leave as is and fix that in a separate PR for your and existing code parts. I would rather not recommend that you fix existing stuff in this PR